lang/src/org/partiql/lang/errors/ErrorCode.kt

```diff
@@ -552,6 +552,16 @@ enum class ErrorCode(private val category: ErrorCategory,
         LOCATION,
         "LIMIT must not be negative"),
 
+    EVALUATOR_DIVIDE_BY_ZERO(
+        ErrorCategory.EVALUATOR,
+        LOCATION,
+        "/ by zero"),
+
+    EVALUATOR_MODULO_BY_ZERO(
+        ErrorCategory.EVALUATOR,
+        LOCATION,
+        "% by zero"),
```

Comment:
I'm not certain there's a good reason to have a separate error code for modulo by zero.

For precedence, the JVM's error message for `1 % 0` is: `java.lang.ArithmeticException: / by zero`

Reply:
I think it's worth it to have a better error message.